### ISSUE TITLE
Create script to fetch the timetable information from the FTP server

### ISSUE THREAD

**Issue by jwestw** [ONSdigital/SDG_11.2.1#60]
Before we start this, we should estimate how long and how much effort it would take to complete. An alternative is just to go through a manual process using FileZilla and a zip utility. The manual process with some instructions in the user docs is MVP. 

The problem: We need to access the TNDS data at ftp.tnds.basemap.co.uk . I don't know how often the data is updated but I suspect it will need a regular re-downloading, hence, an auto-download feature would be a **nice to have**. 

We have an FTP module already so can this be adapted to fetch data from the Travel Line (TNDS) FTP server without too much extra work?

- I have the username and password already. 
- the files are all zipped (so if we were to implement and auto-download feature, then we would need to adapt the `_import_extract_delete_zip` function.  
- File format is in xml

**Comment by james-westwood** [comment]
Note: for longer-term future user purposes, an automatic download, by region, might be best. The reason is the unzipped XML files amount to about 25GB data, so we probably do not want to store those on our own FTP server, or expect future users to do that.

**Comment by jwestw** [comment]
Based on the meeting we had this morning, the DSC team suggest perhaps using a Google Cloud hosted SQL server with Big Query.

**Comment by nkshaw23** [comment]
Closed by #257 which creates an auto download flag to fetch latest timetable data.
